### ISSUE TITLE
cant move the counter

### ISSUE THREAD

**Issue by chingchong69** [ericBG/AnyfinCalculator#9]
the counter is just a big fat block in the centre of the board and it doesn't unlock with the rest of the overlay

**Comment by ericBG** [comment]
Yes, that was a planned feature. However, it cannot be integrated with the
normal unlock mechanism - that's only for HDT only things. I'll work on it
soon, but I am on holiday right now

**Comment by MT-Arnoldussen** [comment]
Was about to report the same issue.

If you can't get it to move draggable, can you at least put it somewhere it doesn't block the attack/health of an enemy minion? (when you have the time ofc :))

This tracker is really awesome otherwise, and thank you for that!

**Comment by ericBG** [comment]
Sorry I've been away for a while, will get working on it when I get home on
the train today. Sorry to everyone for forgetting about this addon!

**Comment by Devastatin** [comment]
There is an XML file that allows you to manually "move" the starting location of the icon. This could be a workaround for those who use the calculator. 

I think its anyfin.xml and its in one of the starting locations of HDT.

**Comment by kkboxop** [comment]
@Devastatin would you elaborate more? I am trying to modify .dll file itself, am I doing wrong? 
@ericBG would you help me please? I am not sure why but modify this line of code didn't change anything
`mc:Ignorable="d" Height="100" Width="145"`
or
`Source="./Images/swamp.png" Margin="10,26,80,26"`

**Comment by Devastatin** [comment]
@kkboxop I didn't modify the dll. There was an xml file on my computer that I found and modified. 

Look in the installation directory of HST,  right below where you downloaded the plugin.  And please note,  my computer died and I'm doing this from memory.  Look for anyfin.xml (or something similar). I can guarantee it's an xml file
